test/python/test_quantumprogram.py

```diff
@@ -1442,16 +1442,17 @@ def swap(qc, q0, q1):
         self.assertEqual(result.get_counts("swapping"),
                          {'010000': 1024})
 
-    def test_offline(self):
-        import string
-        import random
-        qp = QuantumProgram()
-        FAKE_TOKEN = 'this_token_is_not_going_to_be_sent_nowhere'
-        FAKE_URL = 'http://{0}.com'.format(
-            ''.join(random.choice(string.ascii_lowercase) for _ in range(63))
-        )
-        # SDK will throw ConnectionError on every call that implies a connection
-        self.assertRaises(ConnectionError, qp.set_api, FAKE_TOKEN, FAKE_URL)
+    # def test_offline(self):
```

Comment:
Remove dead code or disable the test explicitly with [`unittest.skip`](https://docs.python.org/3.6/library/unittest.html#unittest.skip)